Objects/listobject.c

```diff
@@ -571,18 +572,22 @@ list_repeat(PyListObject *a, Py_ssize_t n)
     }
     else {
         PyObject **src = a->ob_item;
-        PyObject **src_end = src + Py_SIZE(a);
         while (src < src_end) {
```

Comment:
You got rid of `src_end` but still use it...

Reply:
Updated.